src/partials/contacts.html

```diff
@@ -0,0 +1,22 @@
+<contacts class="container">
+    <div class="contacts__section">
+        <div class="contacts__box-text1">
+            <p class="contacts-text1">Registration for a <span class="contacts-text-color">free</span> phone <span class="contacts-text-color">consultation</span> on the nuances of English UPE
+            </p>
+        </div>
+        <div  class="contacts__box-text2">
+            <p class="contacts-text2">
+                Leave a request for participation in a <span class="contacts-text-color">free lesson</span>.
+                A consultant will contact you and answer your questions
+            </p>
+        </div>
+        <div class="contact-tabel-size">
+            <form>
+                <label>
+                    <input class="contacts-tabel-name" type="name" name="name" placeholder=" Name*" />
```

Comment:
W przypadku formularza musisz dodać minimalną walidację danych wejściowych. Aby to zrobić, możesz użyć atrybutu pattern.

Reply:
W przypadku wprowadzenia nieprawidłowych danych pole powinno być podświetlone na czerwono. Walidacja nie powinna być podświetlona, ​​dopóki użytkownik nie wprowadzi danych. Warto codać też  required